ieeeuoft/newhacks-site-2022
# NH-23: Allow Test User Access to HSS

## Overview

- Resolves #NH-23
- added the ability for test users to place orders and use HSS


## Unit Tests Created

- added 1 test in hardware/test_api for the group


## Steps to QA

- try to use the hss with the "HSS Test Users" group attached to your user, all actions should work
